src/components/EditForm/EditForm.js

```diff
@@ -0,0 +1,34 @@
+import { useState, useContext } from "react";
+import { AgencyContext } from "../../context/AgencyContext";
+import DataForm from "../DataForm/DataForm";
+import "./EditForm.css";
+
+export default function EditForm() {
+  const { submitData, state, onChangeHandler } = useContext(AgencyContext);
```

Comment:
- This is the place to call `useForm`
- Then you can the agency to useForm as an initial state

Consider passing [defaultValues when you call useForm](https://react-hook-form.com/api/useform)